docs/getting_started.md

```diff
@@ -129,11 +129,7 @@ $ brew install cmake bison libffi mcpp pkg-config
 $ brew reinstall gcc
 $ brew link bison --force
 $ brew link libffi --force
-
-$ git clone https://github.com/placidex/homebrew-souffle
-$ cd homebrew-souffle/Formula
-$ git checkout souffle-2.3
-$ brew install --build-from-source ./souffle.rb
+$ brew install souffle-lang/souffle/souffle
```

Comment:
simplified since https://github.com/souffle-lang/homebrew-souffle/pull/13 is merged 😃